Lib/_collections_abc.py

```diff
@@ -793,6 +793,7 @@ def __isub__(self, it):
 
 ### MAPPINGS ###
 
+TPFLAGS_MAPPING = 1 << 6
```

Comment:
If this is public and can be relied upon, it needs to be added to the docs for collections.abc

Reply:
This should be internal. I'll hide it.